src/components/Option/index.js

```diff
@@ -94,9 +95,12 @@ class OptionItem extends Component {
     }
 
     renderRightIcon() {
-        const { name, currentValues, icon, iconPosition } = this.props;
+        const { name, currentValues, icon, iconPosition, activeOptionName } = this.props;
         const hasRightIcon = !!(icon && iconPosition === 'right');
         if (currentValues && currentValues.includes(name)) {
+            if (activeOptionName === name) {
+                return <StyledUncheckIcon />;
+            }
```

Comment:
this will affect the piclklist too, how we can avoid this?

Reply:
I think we could pass down the multiple prop through context and then only render the uncheck icon when is multiple thus simple usages like in Picklist will be not affected

Reply:
Done